packages/components/src/components/PipelineRuns/PipelineRuns.js

```diff
@@ -97,51 +97,59 @@ const PipelineRuns = ({
     }
   });
 
-  const pipelineRunsFormatted = pipelineRuns.map(pipelineRun => ({
-    id: `${pipelineRun.metadata.namespace}:${pipelineRun.metadata.name}`,
-    name: (
-      <Link
-        to={createPipelineRunURL({
-          namespace: pipelineRun.metadata.namespace,
-          pipelineRunName: createPipelineRunDisplayName({
-            pipelineRunMetadata: pipelineRun.metadata
-          })
-        })}
-      >
-        {pipelineRun.metadata.name}
-      </Link>
-    ),
-    pipeline: !hidePipeline && (
-      <Link
-        to={createPipelineRunsByPipelineURL({
-          namespace: pipelineRun.metadata.namespace,
-          pipelineName: pipelineRun.spec.pipelineRef.name
-        })}
-      >
-        {pipelineRun.spec.pipelineRef.name}
-      </Link>
-    ),
-    namespace: !hideNamespace && pipelineRun.metadata.namespace,
-    status: (
-      <div className="definition">
-        <div
-          className="status"
-          data-status={getStatus(pipelineRun).status}
-          data-reason={getStatus(pipelineRun).reason}
-        >
-          <div className="status-icon">
-            {getPipelineRunStatusIcon(pipelineRun)}
+  const pipelineRunsFormatted = pipelineRuns.map(pipelineRun => {
+    const { namespace, annotations } = pipelineRun.metadata;
+    const pipelineRunName = createPipelineRunDisplayName({
+      pipelineRunMetadata: pipelineRun.metadata
+    });
+    const pipelineRefName = pipelineRun.spec.pipelineRef.name;
+    const pipelineRunType = pipelineRun.spec.type;
+    const { reason, status } = getStatus(pipelineRun);
+    const statusIcon = getPipelineRunStatusIcon(pipelineRun);
+    const pipelineRunStatus = getPipelineRunStatus(pipelineRun, intl);
+    const url = createPipelineRunURL({
+      namespace,
+      pipelineRunName,
+      annotations
+    });
+    return {
+      id: `${namespace}:${pipelineRunName}`,
+      name: <Link to={url}>{pipelineRunName}</Link>,
+      pipeline:
+        !hidePipeline &&
+        (pipelineRefName ? (
+          <Link
+            to={createPipelineRunsByPipelineURL({
+              namespace,
+              pipelineName: pipelineRefName
+            })}
+          >
+            {pipelineRefName}
+          </Link>
+        ) : (
+          ''
+        )),
+      namespace: !hideNamespace && namespace,
+      status: (
+        <div className="definition">
```

Comment:
Can you rebase and make sure this still works with long status messages and the change from https://github.com/tektoncd/dashboard/pull/841? The icon and text should appear on the same line, and long text should not wrap below the icon, e.g.
![image](https://user-images.githubusercontent.com/2829095/70926162-84f85480-2024-11ea-99d4-e10dcc872cae.png)
and
![image](https://user-images.githubusercontent.com/2829095/70926183-8e81bc80-2024-11ea-8be6-135474761944.png)